[WIP] Add a blog post for "Running a Great Socratic Seminar" (#47)
BitDevsNYC/BitDevsNYC.github.io: _posts/2020-02-14-running-a-great-socratic-seminar.md

The [Optech newsletter](https://bitcoinops.org/en/newsletters/) is also a good
source of relevant content.

Tailor the content to what the attendees will find interesting. It's OK if you
aren't having the exact same discussion that another city is having. Maybe your
group is more excited about privacy because of the part of the world you are
in, or maybe you are worried about regulation because you live in a financial
capital. Make it your own and tailor it to the people that show up
consistently.

### Preparation

With content selected, it's time to prepare yourself to lead the discussion.
First, read and highlight the material. There are highlighting plugins for all
browsers. Use highlights to emphasize important content on the page -- they're
a good mnemonic device for the host and they help drive discussion. That said,
be careful not to use the highlights as a crutch -- if you exclusively read
word-for-word from the links and do not engage the crowd, you'll lose their
attention and discussion will fall flat.

Generally, preparing slides isn't useful or necessary -- it takes a long time
to prepare, and people want to see the source material anyway. 

> Arrange the tabs in your browser from left to right, starting with the things

Reviewer: Could probably drop "from left to right" to leave it up to the reader to choose whatever ordering they find most natural (e.g. a Hebrew reader might prefer right to left).

---
layout: blog
type: blog
title: "Running a Great Socratic Seminar"
permalink: /running-a-great-socratic-seminar/
---

People often ask: why run a meetup? Isn't it a lot of work? What is the incentive if you aren't being paid?

Shaping a local community around shared values and common interests is quite rewarding; the friendships you build and the impact you have on one another span a lifetime--you cannot put a dollar value on the collective experiences, cultivated connections and acquired knowledge. 

> That said, wanting to run a meetup and actually running one are two very different things. Fostering an unwelcoming, stale environment that does not encourage high quality discussion and regular attendance should be avoided at all costs. Be prepared to dedicate a non-trival amount of time aggregating, digesting and preparing content in a stimulating way. 

Reviewer 2: non-trival -> non-trivial

Reviewer 3: ```suggestion
That said, wanting to run a meetup and actually running one are two very different things. Fostering an unwelcoming, stale environment that does not encourage high quality discussion and regular attendance should be avoided at all costs. Be prepared to dedicate a non-trivial amount of time aggregating, digesting and preparing content in a stimulating way. 
```

> This blog post focuses on a discussion-focused event format that is derivative of the "socratic seminar,[LINK]" a meetup style that we have had a lot of success with in the tri-state (NY/NJ/CT) Bitcoin community. The community driven nature of these events stems from the desire to not only encourage participation from attendees and keep content fresh, but to also break reliance on outside presenters for content.  

Reviewer 2: with in the tri-state -> within the tri-state

Reviewer 3: ```suggestion
This blog post describes a discussion-focused event format that is derivative of the "socratic seminar,[LINK]" a meetup style that we have had a lot of success within the tri-state (NY/NJ/CT) Bitcoin community. The community driven nature of these events stems from the desire to not only encourage participation from attendees and keep content fresh, but to also break reliance on outside presenters for content.  
```

The format of the event itself is described here [LINK]. The question becomes: how can you successfully prepare for and operate a meetup that satisfies the requirements of a socratic seminar? 

> Presuming you already have a meetup page, the first step is finding event space. Depending on your locale, finding a host can be of varying difficulty. The main requirements for a location are that it is quiet, has ample seating and a projector; as your group grows in size you may also want some kind of microphone setup. Don't let the lack of microphone or projector deter you from all spaces as you can bring your own if you have no other choice.  

Reviewer 2: grows in size -> grows in size,

Reviewer 3: ```suggestion
Presuming you already have a meetup page, the first step is finding event space. Depending on your locale, finding a host can be of varying difficulty. The main requirements for a location are that it is quiet, has ample seating and a projector; as your group grows in size, you may also want some kind of microphone setup. Don't let the lack of microphone or projector deter you from all spaces as you can bring your own if you have no other choice.  
```

If an obvious host is not immediately apparent, here are some connections you may want to seek out:
- Where are other bitcoin/crypto meetups being held in your city? Attend those events and speak to the organizers and hosts, see if they are interested in hosting a socratic seminar. 
- Ask your members--many may have an available space at their office or will have connections to space in your city.
- Local universities. Reach out to any crypto/bitcoin clubs and see if they can book space on their campus.
- Contact crypto/bitcoin companies in your city and see if they are interested in hosting.
- Coworking spaces often allow members to book meeting rooms. 
- Local libraries may have rooms you can book. 
- Hacker spaces are often amenable to technically-focused discussion events.

> Once you have arranged a host, you'll want to promote the event. You only need a handful of people to host a socratic seminar so don't stress over having a massive turnout (especially in the beginning). If you schedule the events regularly (e.g. once a month) your attendance will trend upwards over time. Consistent scheduling is a key component to running a good meetup--become a staple of your local crypto community. A common concern surrounds the possibility of there being no enthusiasts in your local community. While theoretically possible, do not lean into that trap. If you can create a space for thoughtful and advanced discussion and you will likely be surprised by who shows up. 

Reviewer 2: seminar so -> seminar, so

Reviewer 3: ```suggestion
Once you have arranged a host, you'll want to promote the event. You only need a handful of people to host a socratic seminar, so don't stress over having a massive turnout (especially in the beginning). If you schedule the events regularly (e.g. once a month) your attendance will trend upwards over time. Consistent scheduling is a key component to running a good meetup--become a staple of your local crypto community. A common concern surrounds the possibility of there being no enthusiasts in your local community. While theoretically possible, do not lean into that trap. If you can create a space for thoughtful and advanced discussion and you will likely be surprised by who shows up. 
```

> Because meetup.com is a watering hole for people interested in crypto, you can expect some natural inflow just by associating your meetup group to crypto-related keywords [LINK TO MEETUP FAQ]. Attending other crypto meetups and sharing your event with members of those groups is another easy way to increase your RSVP count. Don't be afraid of using various online communication channels to announce your event (e.g. Reddit, twitter, etc.) as well.

Reviewer 2: twitter -> Twitter

Reviewer 3: ```suggestion
Because meetup.com is a watering hole for people interested in crypto, you can expect some natural inflow just by associating your meetup group to crypto-related keywords [LINK TO MEETUP FAQ]. Attending other crypto meetups and sharing your event with members of those groups is another easy way to increase your RSVP count. Don't be afraid of using various online communication channels to announce your event (e.g. Reddit, Twitter, etc.) as well.
```

> If you're aware of specific people in a reasonable distance of your meetup who would be strong candidates for participating in discussion, invite them individually! A lot of people do not actively monitor meetup.com so the only way they might learn about your event is if you invite them. Good candidates for these invitations are individuals who are domain experts or who have very specific interests in an area that you would like to cover. Consider also the timing of your events around others in your area. If there are interesting conferences, hackathons, etc. that draw experts, try and plan your event in a way which includes them. Attendees with domain-specific knowledge can be vital to good discussion. Give them a heads up that you would like their participation--people will not naturally come prepared to engage, you have to foster that environment. 

Reviewer 2: people in a reasonable distance -> people within a reasonable distance

Reviewer 2: participating in discussion -> participating in a discussion/the discussion

Reviewer 2: meetup.com so -> meetup.com, so

Reviewer 2: vital to good discussion -> vital to a good discussion

Reviewer 3: ```suggestion
If you're aware of specific people within a reasonable distance of your meetup who would be strong candidates for participating in the discussion, invite them individually! A lot of people do not actively monitor meetup.com, so the only way they might learn about your event is if you invite them. Good candidates for these invitations are individuals who are domain experts or who have very specific interests in an area that you would like to cover. Consider also the timing of your events around others in your area. If there are interesting conferences, hackathons, etc. that draw experts, try and plan your event in a way which includes them. Attendees with domain-specific knowledge can be vital to a good discussion. Give them a heads up that you would like their participation--people will not naturally come prepared to engage, you have to foster that environment. 
```

The next hurdle is content discovery and collation. The materials you select should speak to what you and the attendees will find interesting. While not every topic will be of interest to everyone, try and find things that most people can relate to. Topics we often look at include:
- Critical vulnerabilities in popular hardware and software
- Network statistics and phenomena (e.g. oscillations in the mempool/difficulty/UTXO set, reorgs, 51% attacks, double spends)
- Mailing list posts with interesting proposals or critiques
- Pull requests from popular repositories that show substantial changes in behavior or architecture as well as improve security or performance.
- Research papers with substantial insights into topics relevant to bitcoiners

> - Blog posts that investigate topics of privacy, software development, cryptography and more. 

Reviewer 2: Your selection of content needs to reflect your audience. It's OK if you aren't having the exact same discussion that another city is having. Maybe your group is more excited about privacy because of the part of the world you are in, or maybe you are worried about regulation because you live in a financial capital. Make it your own and tailor it to the people that show up consistently.

Reviewer 3: ```suggestion
- Blog posts that investigate topics of privacy, software development, cryptography and more. 

Your selection of content needs to reflect your audience. It's OK if you aren't having the exact same discussion that another city is having. Maybe your group is more excited about privacy because of the part of the world you are in, or maybe you are worried about regulation because you live in a financial capital. Make it your own and tailor it to the people that show up consistently.
```

> Thankfully there are many socratic style meetups worldwide which post their content for public consumption; going through their recent events will give you a lots of materials to choose from [LINK][LINK[LINK]. Once you've collated the material you would like to cover and are preparing it for discussion, we recommend highlighting important content. Highlights are a good mnemonic device for the moderator and they help drive discussion from the crowd. Popular browser platforms have plugins that allow you to highlight copy on the page. 

Reviewer 2: give you a lots of materials -> give you lots of/a lot of

Reviewer 3: ```suggestion
Thankfully there are many socratic style meetups worldwide which post their content for public consumption; going through their recent events will give you a lot of materials to choose from [LINK][LINK[LINK]. Once you've collated the material you would like to cover and are preparing it for discussion, we recommend highlighting important content. Highlights are a good mnemonic device for the moderator and they help drive discussion from the crowd. Popular browser platforms have plugins that allow you to highlight copy on the page. 
```

Be careful not to use the highlights as a crutch--if you exclusively read word-for-word from the links and do not engage the crowd when presenting the material, you will lose their attention and discussion may fall flat. In general we do not recommend regurgitating the content into a powerpoint--people want to see the source material and you will want to avoid bloating your preparation time generating slides. 

In terms of ordering the content, we recommend you arrange the tabs in your browser from left to right, starting with the things you would like to cover first. Avoid starting the event with very esoteric/detailed subject matter. Find something that has relevance for the whole group and is ripe for discussion. This will warm up the room and get everyone engaged. From here you can dig more deeply into highly technical protocol-specific subject matter (e.g. pull requests, BIPs, BOLTs, etc..). Once through some of the content with a high degree of technicality, you can soften the end of the meetup by covering/demoing fun projects (e.g. lightning games). We recommend using a browser plugin to save your tabs in the event your browser or browser crashes, device loses power etc.

> With links selected, tabs ordered and content highlighted, it is time to prepare yourself to lead discussion. Reading the material is the first step. Once you have read and highlighted the material, you should practice explaining it--this will uncover holes in your knowledge that were not previously apparent. The more you prep, the more enjoyable the event will be, both for you and the attendees. Ultimately, your goal is to have just enough of an understanding of the material so that you can contextualize the content in a way which avoids the attendees feeling completely lost. 

Reviewer 2: yourself to lead discussion -> yourself to lead the discussion

Reviewer 3: ```suggestion
With links selected, tabs ordered and content highlighted, it is time to prepare yourself to lead the discussion. Reading the material is the first step. Once you have read and highlighted the material, you should practice explaining it--this will uncover holes in your knowledge that were not previously apparent. The more you prep, the more enjoyable the event will be, both for you and the attendees. Ultimately, your goal is to have just enough of an understanding of the material so that you can contextualize the content in a way which avoids the attendees feeling completely lost. 
```

> You will have to balance this form of preparation with your own schedule, as there is always more to learn. Do your best to prepare, but do not over extend yourself. It is always good to remind yourself and attendees that the material is presented on a best effort basis--you are a conduit, not the source itself. Be open to making mistakes and encouraging the crowd to participate. Treat the event like the review process for open source software: stand on the shoulder of giants, don't box yourself into an island of "expertise" at the front of the room.

Reviewer 2: over extend -> overextend

Reviewer 3: ```suggestion
You will have to balance this form of preparation with your own schedule, as there is always more to learn. Do your best to prepare, but do not overextend yourself. It is always good to remind yourself and attendees that the material is presented on a best effort basis--you are a conduit, not the source itself. Be open to making mistakes and encouraging the crowd to participate. Treat the event like the review process for open source software: stand on the shoulder of giants, don't box yourself into an island of "expertise" at the front of the room.
```

> In terms of presupposed knowledge, it is our recommendation that you communicate to your members that the basics of Bitcoin will not be explained at each event (e.g. what a hash function is). That said, when the ecosystem introduces novel concepts/schemes/mechanisms to the Bitcoin community, make sure to explain them. For example, if you are going to explain a new opcode, put the explanation in the context of preexisting opcodes and why it may or may not be an improvement to the protocol. That may require you to explain how opcodes work. Always keep in mind the perspective of the meetup members who come to each event--they play a key role in bringing life to the meetup. Find a way to quickly explain recurring themes in snippets so you do not bore people who regularly attend the events. 

Reviewer 2: Preparation is important, but it's also effective when you say at the outset of a topic that you don't know a subject well and could use some help explaining it to the group. This encourages audience participation and also allows attendees to feel smart as opposed to constantly feel the information is only coming from the hosts.

Reviewer 3: ```suggestion
In terms of presupposed knowledge, it is our recommendation that you communicate to your members that the basics of Bitcoin will not be explained at each event (e.g. what a hash function is). That said, when the ecosystem introduces novel concepts/schemes/mechanisms to the Bitcoin community, make sure to explain them. For example, if you are going to explain a new opcode, put the explanation in the context of preexisting opcodes and why it may or may not be an improvement to the protocol. That may require you to explain how opcodes work. Always keep in mind the perspective of the meetup members who come to each event--they play a key role in bringing life to the meetup. Find a way to quickly explain recurring themes in snippets so you do not bore people who regularly attend the events. 

Preparation is important, but it's also effective when you say at the outset of a topic that you don't know a subject well and could use some help explaining it to the group. This encourages audience participation and also allows attendees to feel smart as opposed to constantly feel the information is only coming from the hosts.
```

Reviewer 4: Hmm, I don't love the wording of this. "Feeling smart" seems out of character with the rest of the post. IMO what is most important is that people feel like that their opinion is welcome, not that it has any specific quality of being smart, not smart etc.

> Once you have established context, discussion can begin. In some cases discussion will commence unprompted, although having some prepared discussion questions can force the attendees to think about the material in way that encourages participation. Not every topic requires discussion, so don't avoid subject matter you think is important because it may not be not be interesting for discussion. You're telling a story, some topics are minor plot devices, others are more central elements of "the conversation."

Reviewer 2: not be ~not be~

Reviewer 3: ```suggestion
Once you have established context, discussion can begin. In some cases discussion will commence unprompted, although having some prepared discussion questions can force the attendees to think about the material in way that encourages participation. Not every topic requires discussion, so don't avoid subject matter you think is important because it may not be interesting for discussion. You're telling a story, some topics are minor plot devices, others are more central elements of "the conversation."
```

> To that end, there are many topics which can lead to circular, tangential or virulent discussion. As the moderator you should not be afraid of cutting off comments which undermine a respectful atmosphere and/or stymie the forward motion of the event. It is our recommendation that you avoid played out debates as well (e.g. block size, segwit), as they are topics which can be endlessly debated with little benfit or novel insight. Lead by example. Set the tone as the host. If you are disrespectful and don't keep discussion focused you can expect the overall quality of the event to be very low. 

Reviewer 2: benfit -> benefit

Reviewer 3: ```suggestion
To that end, there are many topics which can lead to circular, tangential or virulent discussion. As the moderator you should not be afraid of cutting off comments which undermine a respectful atmosphere and/or stymie the forward motion of the event. It is our recommendation that you avoid played out debates as well (e.g. block size, segwit), as they are topics which can be endlessly debated with little benefit or novel insight. Lead by example. Set the tone as the host. If you are disrespectful and don't keep discussion focused you can expect the overall quality of the event to be very low. 
```

Miscellaneously, we recommend:

> - Do not record the event. If you want people to speak freely, do not point the internet's firehose at them. Recording the event will likely lead to self-censorship and limit discussion. 

Reviewer 2: Also, discourage others from taking pictures of other members or presentations. Have some pens and paper available if people feel like they need to write notes.

Reviewer 3: ```suggestion
- Do not record the event. If you want people to speak freely, do not point the internet's firehose at them. Recording the event will likely lead to self-censorship and limit discussion. Also, discourage others from taking pictures of other members or presentations. Have some pens and paper available if people feel like they need to write notes.
```

- Do introductions. In the beginning of the event give folks 10-20 seconds to say their name and mention a topic that interests them. People may announce other events, job listings, tell jokes and share other things relevant to your members. This brings members closer together, and gives people a chance to get to know each other. 

> - Setup a table at the front of the room so you are at the same level as the crowd, this makes you less of an authority and more of a participant in discussion. 

Reviewer 3: ```suggestion
- Setup a table at the front of the room so you are at the same level as the crowd, this makes you less of an authority and more of a participant in discussion. 
- Especially for smaller groups, arrange the seating in a circle, this enables anyone to look each other in the eyes and fosters high quality conversations.
```

- Always bring every possible converter and extension cable you may need (e.g. HDMI extension, VGA to X, DVI to X and HDMI to X)
- Find a place after the event to gather for drinks and food. Post this on the meetup page so people who cannot make the main event can join for the after party.

> - Manage your time. Respect that many people have families or come from far away and need to get home at a reasonable hour.

Reviewer 3: ```suggestion
- Manage your time. Respect that many people have families or come from far away and need to get home at a reasonable hour.
- Mention in the beginning that anyone can leave the conversation at any time, though to do so quiet and respectful of the ongoing discussion. 
```

- Short presentations at the end can be very exciting. We recommend following these rules [LINK] to ensure high quality presentations that are relevant to attendees. 
- Triple check with your event location! Make sure there are no miscommunications on date/time. 
- If you can find a sponsor to provide food, great! But be careful who you get in bed with--we do not recommend taking sponsorship from ICOs and altcoins for obvious reasons. 

> - Have fun! 

Reviewer 2: Maybe another opportunity to reiterate that consistency is what builds an audience. Sticking with a regular meetup while the market is hot and cold reflects the staying power of the meetup.

Reviewer 3: ```suggestion
- Consistency is what builds an audience. Sticking with a regular meetup while the market is hot and cold reflects the staying power of the meetup.
- Have fun! 
```
